src/App.js

```diff
@@ -1,25 +1,34 @@
-import logo from './logo.svg';
-import './App.css';
+import { Redirect, Route, BrowserRouter as Router, Switch } from "react-router-dom";
+import styled from "styled-components";
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+import backgroundImage from "./assets/background-image.jpg";
+import DetailsPage from "./pages/DetailsPage";
+import MainPage from "./pages/MainPage";
+
+const MainWrapper = styled.div`
```

Comment:
In general you want to put styles in a separate file, and importing backgroundImage should not be done like this.

Reply:
And if you look at the background image it is stretched over a height of about 6000px